addons/project/static/src/components/project_stop_recurrence_confirmation_dialog/project_stop_recurrence_confirmation_dialog.js

```diff
@@ -3,12 +3,24 @@
 import { ConfirmationDialog } from '@web/core/confirmation_dialog/confirmation_dialog';
 
 export class ProjectStopRecurrenceConfirmationDialog extends ConfirmationDialog {
-    _continueRecurrence() {
-        if (this.props.continueRecurrence) {
-            this.props.continueRecurrence();
-        }
+    _addressRecurrence() {
+        const targetOption = document.querySelector('.o_radio_input[name="targetOption"]:checked').id;
+        this.props.addressRecurrence(targetOption);
         this.props.close();
     }
 }
 ProjectStopRecurrenceConfirmationDialog.template = 'project.ProjectStopRecurrenceConfirmationDialog';
-ProjectStopRecurrenceConfirmationDialog.props.continueRecurrence = { type: Function, optional: true };
+ProjectStopRecurrenceConfirmationDialog.props = {
+    ...ConfirmationDialog.props,
+    addressRecurrence: { type: Function, optional: true },
+    modeLabel: { type: String, optional: true },
+    targets: { type: Array, optional: true },
+}
+ProjectStopRecurrenceConfirmationDialog.defaultProps = {
+    ...ConfirmationDialog.defaultProps,
+    targets: [
+        ['this', 'this task, but continue the recurrence'],
+        ['subsequent', 'this task, and stop the recurrence'],
+        ['all', 'all tasks, and stop the recurrence'],
```

Comment:
These strings should be translatable. Also, I feel like having `Delete` on one line and then the list of options, while it looks nice in English, might cause problems in other languages where the sentences are structured differently (e.g. languages where the verb goes at the end, like Turkish).